docs/pages/desktop-access/introduction.mdx

```diff
@@ -21,13 +19,17 @@ Desktop Access, you get:
 >
   Only Windows hosts accessible over RDP are supported. Specifically:
 
-  - Windows Server 2012 R2 / Windows 10 or newer
-  - Hosts connected to an Active Directory domain
+| Teleport Version              | Windows Desktop                                           | Active Directory |
+|-------------------------------|-----------------------------------------------------------|------------------|
+| Open source Teleport          | Windows Server 2012 R2 / Windows 10 or newer              | Required         |
+| Cloud and Enterprise Teleport | Windows Server 2012 R2 through 2019 / Windows 10 or newer | Optional         |
```

Comment:
This is not quite right, because enterprise teleport with AD works just fine with WS2022

Reply:
How about I change it to this:

```suggestion
| Cloud and Enterprise Teleport | Windows Server 2012 R2 / Windows 10 or newer | Optional         |
```

Since there's already a warning in getting-started about 2022?

Reply:
Works for me